src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -661,7 +661,7 @@ bool emberAfOperationalCredentialsClusterAttestationRequestCallback(app::Command
     Platform::ScopedMemoryBuffer<uint8_t> attestationElements;
     size_t attestationElementsLen = 0;
     MutableByteSpan attestationElementsSpan;
-    uint8_t certDeclBuf[512];
+    uint8_t certDeclBuf[541]; // Sized to hold the example certificate declaration wtih 100 PIDs. See DeviceAttestationCredsExample
```

Comment:
This needs to be some spec-defined constant, no?  Otherwise it will fail on some other certificate declaration that happens to be bigger than this one but is allowed by the spec.

Reply:
- Make a `kMaxCMSSignedCDMessage` for sure, based on the max size + some margin (I think 541 is close to the max, but not quite. 600 would be a better value with a bit of slack and to match max DER size of DAC/NOC.

Reply:
See above - constant got defined in another header, but wasn't used here before. Updated in next commit to use that constant.